splines/splineFDBase.cpp

```diff
@@ -683,7 +682,7 @@ void splineFDBase::PrepForReweight()
       UniqueSystSplines[iSpline]->GetKnot(iKnot, xPoint, yPoint);
       if (xPoint == -999 || yPoint == -999)
```

Comment:
magic numbers

Reply:
another case where this won't actually do anything since if GetKnot fails it won't just return -999.